metadata-ingestion/src/datahub/ingestion/source/powerbi/rest_api_wrapper/data_resolver.py

```diff
@@ -83,7 +91,21 @@ def __init__(
         client_secret: str,
         tenant_id: str,
         metadata_api_timeout: int,
+        environment: str = "commercial",
     ):
+        self._environment = environment
+        self._urls = (
+            self.GOVERNMENT_URLS
+            if environment == "government"
+            else self.COMMERCIAL_URLS
+        )
+
+        self.SCOPE = self._urls["SCOPE"]
+        self.MY_ORG_URL = self._urls["MY_ORG_URL"]
+        self.BASE_URL = f"{self.MY_ORG_URL}/groups"
+        self.ADMIN_BASE_URL = f"{self.MY_ORG_URL}/admin"
+        self.AUTHORITY = self._urls["AUTHORITY"]
```

Comment:
Instance variables should be lowercase, and prefixed with underscore since they're private. `_urls` should probably only be a local variable since it's only used here.